### DISCUSSION TITLE
Adding WSL to windterm Shell Sessions

### DISCUSSION THREAD

**Discussion by nyxonman** [kingToolbox/WindTerm#1189] [Q&A] [answered]
Hi,

I have recently installed WSL2 in my machine but it does not automatically appear on the shell sessions. How may I add the WSL session to the windterm?

![image](https://user-images.githubusercontent.com/41434353/188840819-79f5a09b-fd7b-4e88-bf2a-33ae286384a5.png)

**Comment by nyxonman** [comment, marked as answer]
nvm i found the way under shell session settings. Thanks

**Comment by wt159** [comment]
你好，我想问一下是怎么添加的呢？谢谢。

**Reply by JeEisenberg** [reply]
![image](https://github.com/kingToolbox/WindTerm/assets/94214876/5cbffc2e-665b-41cd-b3a2-950eef4e7593)
就像这样,之后就可以了(在shell处选择自定义然后输入wsl),点击确定就可以了
